gum/backend-qnx/gumprocess-qnx.c

```diff
@@ -169,12 +168,12 @@ gum_qnx_enumerate_ranges (pid_t pid,
                           GumFoundRangeFunc func,
                           gpointer user_data)
 {
-
   gchar * as_path;
   gint fd, res;
   gboolean carry_on = TRUE;
-  procfs_mapinfo membufs[512];
-  gint num_membufs;
+  procfs_mapinfo * mapinfos;
+  gint num_mapinfos;
+  procfs_debuginfo * debuginfo = malloc (sizeof(procfs_debug) + 0x100);
```

Comment:
Should be a space after `sizeof`, and also it would be good to move the call out of the list of declarations. (We tend to only do simple assignments there, and no declaration after statement.)